beaker-tests/Sanity/copr-cli-basic-operations/runtest-exclusivearch.sh

```diff
@@ -33,7 +33,8 @@ rlJournalStart
 
         # This is a more complicated package with `BuildArch: noarch` and
         # ExclusiveArch for subpackages. Test that we don't fail while parsing it
-        rlRun "copr-cli build-distgit ${NAME_PREFIX}ExclusiveArch --name procyon"
+        # TODO: we fail the build actually, https://github.com/fedora-copr/copr/issues/2870
+        #rlRun "copr-cli build-distgit ${NAME_PREFIX}ExclusiveArch --name procyon"
```

Comment:
The whole PR looks good to me but since it is not a release blocker, we should probably fix the TODO before merging.

Reply:
Maybe by changing the package that we test or uncommenting this line and commenting out the 

```bash
chroots+=" --chroot fedora-$FEDORA_VERSION-ppc64le"
```

That way we can test that at least some chroots were skipped.

Reply:
The package is noarch, and still using `ExclusiveArch:  %{java_arches} noarch`.  In this case I think that the intention is not to ship on i386, but not sure.  Java folks though, I bet, don't want to [skip builds on ppc64le](https://koji.fedoraproject.org/koji/buildinfo?buildID=2226538) :-/

Reply:
> That way we can test that at least some chroots were skipped.

I mean, this package is not supposed to be skipped at build time on all Fedora Copr-supported architectures.

Reply:
What about now?